test/integration/runner_task_test.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+require 'kubernetes-deploy/runner_task'
+class RunnerTaskTest < KubernetesDeploy::IntegrationTest
+  def test_works
+    deploy_fixtures("hello-cloud", subset: ["template-runner.yml"])
+
+    task_runner = KubernetesDeploy::RunnerTask.new(
+      context: KubeclientHelper::MINIKUBE_CONTEXT,
+      namespace: @namespace,
+    )
+
+    task_runner.run(
+      task_template: 'hello-cloud-template-runner',
+      command: nil,
+      args: ['a']
```

Comment:
The entrypoint for `hello-world:latest` is `/hello` and it's still unclear what it would print with `a` as an argument.